docs/reference.rst

```diff
@@ -146,3 +146,51 @@ Example
 
 
 NOTE: at the moment, it is not possible to mix packages and slices in the same stage-packages field.
+
+
+Rockcraft commands
+------------------
+Lifecycle commands
```

Comment:
can you screenshot what this new section looks like, rendered? (You might need to update your doc env, there's a new theme)

Reply:
Sure thing!

Reply:
Thanks! Is there a particular reason why you chose bulleted lists? I think the docs might flow better if we use regular paragraphs for each section, even if they're just a single sentence.
I understand that this is an initial pass to have "something" there, so thanks for doing this regardless ;)

Reply:
No particular reason.  I agree that paragraphs would be better.  I updated the docs and the rendered screenshot.